app/assets/javascripts/html5loc_ajax.js

```diff
@@ -0,0 +1,21 @@
+function onSuccess(pos) {
+  const { latitude, longitude } = pos.coords;
+
+  $.post('/posts/update_location', { latitude, longitude }).done(res =>
+    console.log(`posted successfully`)
```

Comment:
remove

Reply:
leaving for debugging after offline discussion